scripts/main.js

```diff
@@ -0,0 +1,326 @@
+    var material = new THREE.MeshPhongMaterial({color: 0x0066CC});
+    var cube = new THREE.Mesh(geometry, material);
+    scene.add(cube);
+
+
+    geometry = new THREE.PolyhedronGeometry(verticesOfCube, indicesOfFaces, 1.001 - 0.14, 0);
+    material = new THREE.MeshPhongMaterial({color: 0xFFFFFF, wireframe: true});
+    cube = new THREE.Mesh(geometry, material);
+    scene.add(cube);
+
+    var directionalLight = new THREE.DirectionalLight(0xFFFFFF, 0.5);
+    directionalLight.position.x = 6;
+    directionalLight.position.y = 8;
+    directionalLight.position.z = 8;
+    scene.add(directionalLight);
+
+    directionalLight = new THREE.DirectionalLight(0xFFFFFF, 0.5);
+    directionalLight.position.x = -6;
+    directionalLight.position.y = -8;
+    directionalLight.position.z = -8;
+    scene.add(directionalLight);
+
+    // light
+    var ambientLight = new THREE.AmbientLight(0xFFFFFF, 0.8);
+    scene.add(ambientLight);
+}
+
+
+function init() {
+    var scene = new THREE.Scene();
+    var camera = new THREE.PerspectiveCamera(75, window.innerWidth / window.innerHeight, 0.1, 1000);
+
+    var renderer = new THREE.WebGLRenderer({antialias: true});
+    renderer.setSize(window.innerWidth, window.innerHeight);
+
+    document.body.appendChild(renderer.domElement);
+    var controls = new OrbitControls(camera, renderer.domElement);
+    // controls.autoRotate = true;
+    // controls.autoRotateSpeed = 3;
+    var pmremGenerator = new THREE.PMREMGenerator(renderer);
+
+    pmremGenerator.compileEquirectangularShader();
+
+    window.addEventListener("resize", function () {
+        var width = window.innerWidth;
+        var height = window.innerHeight;
+        renderer.setSize(width, height);
+        camera.aspect = width / height;
+        camera.updateProjectionMatrix();
+    });
+
+
+    addPolyhedron(scene);
+    camera.position.z = -2;
+    camera.position.y = 1.1;
+    camera.rotation.y = 3.14;
+    camera.rotation.x = 0.5;
+
+    return [scene, renderer, camera, controls]
+}
+
+
+
+
+window.onload = function() {
+    const FizzyText = function () {
+        this.particles = "0";
+        this.color = [255, 255, 0]; // RGB array
+        this.pointWidth = 1;
+
+        this.time = 0;
+        this.mode = "offline";
+
+        this.drawCube = true;
+        this.play = true;
+        this.turnOn = false;
+        this.currentSave = fileGet("/lib/saves/names.json")["default_name"];
+
+        this.reset_defaults = function () {
+            for (var i = 0; i < gui.__controllers.length; i++) {
+                gui.__controllers[i].setValue(gui.__controllers[i].initialValue);
+            }
+        };
+
+        this.restart = function () {
+            FrameId = 2;
+            speed = 1;
+            fizzyText.play = true;
+            arrayOfPoints = [];
+
+            // clear scene from points
+            while (scene.children.length > 5) {
```

Comment:
Will you have to update this number each time you add some new objects (like cameras, lights, etc) to the simulation? Is there a way to check the condition you actually want to check (`are there any objects with the name "point"`, as far as I'm concerned)?